editor/src/clj/editor/editor_extensions/ui_docs.clj

```diff
@@ -58,24 +87,20 @@
   {:alignment [:top-left :top :top-right :left :center :right :bottom-left :bottom :bottom-right]
    :padding [:small :medium :large]
    :spacing [:small :medium :large]
-   :text_alignment [:left :center :right :justify]
-   :icon_name [:open-resource :plus :minus :clear]
+   :text-alignment [:left :center :right :justify]
+   :icon-name [:open-resource :plus :minus :clear]
```

Comment:
Is `editor.ui.ICON_NAME.OPEN_RESOURCE` used for both resource and external file fields? Should its name reflect that?

Also, should it be renamed to `editor.ui.ICON`? We don't call the color enum `editor.ui.COLOR_NAME`.